# Default initial positions for manipulator1's ros2_control fake system

initial_positions:
  base_link_j_base_plate: 0.0
  motor_holder_j_side_plate_12: 0.0
  side_plate1_j_side_plate12: 0.0
